doc/ref_model/chapters/chapter04.md

```diff
@@ -600,7 +600,28 @@ These flavors, similarly to other flavours, should be supported for three CNTT r
 <a name="4.3"></a>
 ## 4.3 Networking
 
-This is a placeholder for Cloud Infrastructure infrastructure networking information that is common to all Reference Architectures.
+The Cloud Infrastructure Networking Reference Model is an essential foundation that governance all Reference Architectures and Cloud Infrastructure implementations to enable a continuous evolution and migration from current single Infrastructure as a Service (IaaS) based virtualization instances with Virtual Machines (VM) into tomorrow’s multitude of Cloud Native Container as a Service (CaaS) based virtualization instances.  
+
+To retain the Cloud paradigms of automation, scalability and usage of shared hardware resources when introducing CaaS instances it is necessary to enable an ability to co-deploy multiple simultaneous IaaS and CaaS instances on a shared pool of hardware resources.
+
+Compute and Storage resources are rarely shared in between IaaS or CaaS instances, but the underpinning networking most commonly implemented with Ethernet and IP, must be shared and managed to have a shared pool of hardware resources.
```

Comment:
```suggestion
Compute and Storage resources are rarely shared in between IaaS or CaaS instances, but the underpinning networking, most commonly implemented with Ethernet and IP, must be shared and managed as a shared pool of hardware resources.
```

Reply:
@pgoyal01 Updated the textual sugested changes, and added some clarifications on sharing of underlay networking resources